core/azure-core/src/main/java/com/azure/core/polling/package-info.java

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * Package containing API for Long Running Operations.
+ */
+package com.azure.core.polling;
```

Comment:
Let's moving the polling package within the util package.

Reply:
resolved.